vms/avm/index_test.go

```diff
@@ -437,10 +258,6 @@ func buildPlatformUTXO(utxoID avax.UTXOID, txAssetID avax.Asset, addr ids.ShortI
 	}
 }
 
-func signTX(codec codec.Manager, tx *txs.Tx, key *secp256k1.PrivateKey) error {
-	return tx.SignSECP256K1Fx(codec, [][]*secp256k1.PrivateKey{{key}})
-}
```

Comment:
This was just inlined where it was called before... It's a one-liner either way.